### ISSUE TITLE
Error: Building package openssl-windows:x86-windows

### ISSUE THREAD

**Issue by mattlathrop** [microsoft/vcpkg#7056]
It seems http://www.nasm.us/pub/nasm/releasebuilds/2.14/win32/nasm-2.14-win32.zip is offline? Is there a short term workaround?

```
-- Downloading http://www.nasm.us/pub/nasm/releasebuilds/2.14/win32/nasm-2.14-win32.zip...
-- Downloading http://www.nasm.us/pub/nasm/releasebuilds/2.14/win32/nasm-2.14-win32.zip... Failed. Status: 7;"Couldn't connect to server"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:172 (message):

      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "https://user:password@your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  scripts/cmake/vcpkg_find_acquire_program.cmake:262 (vcpkg_download_distfile)
  ports/openssl-windows/portfile.cmake:37 (vcpkg_find_acquire_program)
  scripts/ports.cmake:73 (include)
```

https://www.isitdownrightnow.com/nasm.us.html

**Comment by iamnotamit** [comment]
I can reproduce this issue locally.

Fortunately for me, I have another machine that already had "nasm-2.14-win32.zip" downloaded from earlier when the site was still up so copying that file over fixed things for me.

**Comment by DailyShana** [comment]
www.nasm.us is unreachable in many places: 
https://www.site24x7.com/public/t/results-1562316163264.html

**Comment by yaseenmd** [comment]
Any update on this issue. ? or workaround

**Comment by iamnotamit** [comment]
Looks like [www.nasm.us](www.nasm.us) is back online so I suppose the issue is fixed for now.

**Comment by DailyShana** [comment]
I suggest add a mirror for nasm as #7175 in case of it is down again

**Comment by LilyWangL** [comment]
Thanks for posting this issue. This issue has been fixed in PR #7175. So I am closing this issue for now.
